src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+ * Add support for vertical table rendering
```

Comment:
Add support to display table verticaly?

as your var naming after is display not render

Reply:
Yes @94noni that would be more consistent. Thanks